erica/worker/pyeric/eric_errors.py

```diff
@@ -506,6 +518,9 @@ def _create_transfer_error(rescode, eric_response, server_response, server_err_m
     if rescode == 610101292 and server_err_msg and is_error_in_server_err_msg(server_err_msg.get('NDH_ERR_XML'),
                                                                               '371015213'):
         raise EricAlreadyRevokedError(eric_response, server_response, server_err_msg)
+    if rescode == 610101292 and server_err_msg and is_error_in_server_err_msg(server_err_msg.get('NDH_ERR_XML'),
+                                                                              '371015212'):
```

Comment:
Where does this error code come from? I forgot the part in the documentation...

Reply:
I looked it up in the `VaSt_Fehlercodes.xml` under `Schrittstellenbeschreibungen` in the Eric documentation. :)

Reply:
Thanks. We definitely should keep that in mind when we adapt the documentation in the next days 👍